### PR TITLE
CLDR 44.1 Update - Delimiter, Scripts, Plurals 

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [ ] `ReleaseNotes` has added or is not needed.
* [ ] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [ ] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
* Update `delimiters.jf`, `scripts.jf`, and `plurals.json` files to cldr v44.1
* Fix related test case failures

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

**Review by ehoogerbeets — APPROVED** [review]

File: tools/cldr/genlangscripts.js [hunk 1]
```diff
@@ -46,7 +45,7 @@ process.argv.forEach(function (val, index, array) {
 
 toDir = process.argv[2] || "tmp";
 
-console.log("genlangscripts - generate the localeinfo script.jf files.\n" +
+console.log("genlangscripts - generate the localeinfo scripts.jf files.\n" +
     "Copyright (c) 2013-2018, 2020-2022 JEDLSoft");
```
Comment (ehoogerbeets) [inline tools/cldr/genlangscripts.js:49]: Update the copyright

File: js/test/root/testlocaleinfo.js [hunk 2]
```diff
@@ -11743,8 +11743,8 @@ module.exports.testlocaleinfo = {
         var info = new LocaleInfo("es-ES");
```
Comment (ehoogerbeets) [inline js/test/root/testlocaleinfo.js:11743]: Update the copyright

File: tools/cldr/genlangscripts.js (hunk 1 above)
Comment (gouniLee) [reply tools/cldr/genlangscripts.js]: updated.

File: js/test/root/testlocaleinfo.js (hunk 2 above)
Comment (gouniLee) [reply js/test/root/testlocaleinfo.js:11743]: It has already updated in another PR

**Pull request merged**